arch/arm64/boot/dts/qcom/sdm660-xiaomi-clover-common.dtsi

```diff
@@ -0,0 +1,588 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright (c) 2020, Mark Hargreaves <clashclanacc2602@gmail.com>
+ */
+
+/dts-v1/;
+#include "sdm660.dtsi"
+#include "pm660.dtsi"
+#include "pm660l.dtsi"
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/input/gpio-keys.h>
+
+/delete-node/ &adsp_region;
+/delete-node/ &buffer_mem;
+/delete-node/ &mba_region;
+/delete-node/ &zap_shader_region;
+
+/ {
+
+	qcom,board-id = <11 0>;
+	qcom,msm-id = <324 0>;
+	qcom,pmic-id = <0x0001001b 0x0101011a 0x0 0x0>, <0x0001001b 0x0201011a 0x0 0x0>,<0x0001001b 0x0102001a 0x0 0x0>;
+
+	aliases {
+		serial0 = &blsp1_uart2;
+	};
+
+	chosen {
+		#address-cells = <2>;
+		#size-cells = <2>;
+		ranges;
+
+		stdout-path = "serial0:115200n8";
+
+		framebuffer@9d400000 {
+			compatible = "simple-framebuffer";
+			reg = <0x0 0x9d400000 0x0 (1200 * 1920 * 4)>;
```

Comment:
model, board-id, msm-id, pmic-id (?) should be in board file (dts), not in common dtsi
chosen needs to be in board dts too, screen resolution for framebuffer is hardcoded here, and it is different between clover and clover-plus

Reply:
...or is resolution the same and just size of display is bigger on clover-plus?

Reply:
It is indeed the same